tamr_toolbox/utils/version.py

```diff
@@ -49,8 +53,128 @@ def _as_float(version: str) -> float:
     return (version_split[0] * 1000) + version_split[1] + (version_split[2] / 10)
 
 
+def _get_tamr_versions_from_function_args(*args, **kwargs) -> List[str]:
+    all_args = locals()
+    args = [arg for arg in all_args["args"]]
+    kwargs = list(all_args["kwargs"].values())
+    all_args_parsed = args + kwargs
+    response = []
+
+    # Return the client version (if we can find it)
+    for arg in all_args_parsed:
+        if type(arg) is Client:
+            response.append(current(arg))
+        elif hasattr(arg, "client"):
+            if type(arg.client) is Client:
+                response.append(current(arg.client))
+
+    return response
+
+
+def is_version_condition_met(
+    tamr_version: str,
+    min_version: str,
+    max_version: Optional[str] = None,
+    exact_version: bool = False,
+    raise_error: bool = False,
+) -> bool:
+    """
+    Check Tamr version (and raise error if appropriate).
+
+    Args:
+        tamr_version:
+            The version of Tamr being considered
+        min_version:
+            The minimum version of Tamr to test against
+        max_version:
+            The maximum version of Tamr to test against.
+            Default None, in which case no max version is tested for.
```

Comment:
Small wording change suggestion:
```suggestion
        min_version:
            The minimum allowed version of Tamr
        max_version:
            The maximum allowed version of Tamr
```

Reply:
Agreed